api/src/connection/datasource.ts

```
@@ -10,27 +13,59 @@ const env = process.env;
+      ...commonOptions,
+      type: DbType.BETTER_SQLITE3,
+      database: resolvedPath,
+      foreignKeys: true,
+      enableWAL: true,
+      namingStrategy: new SnakeNamingStrategy(),
+    } as DataSourceOptions;
+  }
+
+  // Handle MySQL or PostgreSQL configuration
+  // Parse the port safely with fallback to default ports based on DB type
   const parsedPort = parseInt(env.TR_DB_PORT, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)